daml-script/daml3/Daml/Script/Questions/Submit/Error.daml

```diff
@@ -22,15 +22,14 @@ data DevErrorType
 
 -- | Additional debugging info optionally provided by the ledger
 data ContractNotFoundAdditionalInfo
-  = NotFound with -- The generic "this contract does not exist"
-      templateIdentifier : Text
+  = NotFound -- The generic "this contract does not exist"
   | NotActive with -- The contract did exist, but has since been archived
-      templateIdentifier : Text
+      contractIdentifier : AnyContractId
```

Comment:
Also, perhaps this should be `contractId`, or the other `contractId`s in SubmitError should be `contractIdentifier`